src/confetti.js

```diff
@@ -228,7 +229,7 @@
 
     var temp = particleCount;
     var fettis = [];
-    var canvas = animationObj ? animationObj.canvas : getCanvas(zIndex);
+    canvas = animationObj ? animationObj.canvas : getCanvas(zIndex);
```

Comment:
This makes `canvas` a global. It will the defined on the global scope (which is bad), and also fails linting. (Note: please make sure that the CI build passes.)